### ISSUE TITLE
UI tests needing a rerun to pass.

### ISSUE THREAD

**Issue by Taknok** [mozilla-mobile/fenix#26984]
### Steps to reproduce

1. Run ui test `deleteMultipleSelectionTest`
2. See first fail and second run suceed.

### Expected behaviour

Pass at first run.

### Actual behaviour

Fail at first run

### Android version

N.A.

### Firefox release type

Firefox

### Firefox version

main

### Additional information

UI tests are failing on first run due to this popup changing the focus. 

![Screenshot_20220910-011045__01](https://user-images.githubusercontent.com/12498020/189457984-3d74e24f-b19e-45eb-b999-9f3094df7d8e.jpg)

This popup is shown when first navigating to an URL. 

> @Taknok Good catch with the tests needing a rerun to pass.
Can you open a new ticket for that? (I can help also)
(the solution is simple, we just need to use `featureSettingsHelper.setTCPCFREnabled(false)`)

_Originally posted by @Mugurell in https://github.com/mozilla-mobile/fenix/issues/21212#issuecomment-1243923644_

Example: ui test `deleteMultipleSelectionTest`

Solution: `featureSettingsHelper.setTCPCFREnabled(false)`

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21567)

**Comment by AaronMT** [comment]
@Taknok This should be resolved today if you pull in latest code off `main`. Please report back if things are not working for you.

**Comment by Taknok** [comment]
Blocked by #27053

**Comment by cpeterso** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1807299

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
